Update lambda.md (#1535)
SumoLogic/sumologic-documentation: docs/integrations/amazon-aws/lambda.md

2. [Configure DataEvents with CloudTrail](https://docs.aws.amazon.com/lambda/latest/dg/logging-using-cloudtrail.html) in your AWS account.
3. Confirm that logs are being delivered to the Amazon S3 bucket.

> 4. Add an [AWS CloudTrail Source](/docs/send-data/hosted-collectors/amazon-aws/aws-cloudtrail-source.md) to Sumo Logic.

Reviewer: Broken link.

Reviewer: This is still broken. Remove.md

5. While configuring the cloud trail log source, following Field can be added in the source:
    * Add an **account** field and assign it a value which is a friendly name / alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried via the “account field”.
5. While configuring the cloud trail log source, the following field can be added to the source:
    * Add an **account** field and assign it a value that is a friendly name/alias to your AWS account from which you are collecting logs. This name will appear in the Sumo Logic Explorer View. Logs can be queried via the “account field”.

<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/AWS-Lambda/Fields.png')} alt="Fields" />

### Collect Amazon CloudWatch Metrics

* Sumo Logic supports collecting metrics using two source types:
    * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)
       or

>     * Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)

Reviewer: Broken link.

Reviewer: Both links are broken.

Author: Resolving/deleting this comment as it's invalid.

> * Configure an [AWS Kinesis Firehose for Metrics Source](/docs/send-data/hosted-collectors/amazon-aws/aws-kinesis-firehose-metrics-source) (Recommended)

Reviewer: Broken link.

Reviewer: Links is still broken.

> * Configure an [Amazon CloudWatch Source for Metrics](/docs/send-data/hosted-collectors/amazon-aws/amazon-cloudwatch-source-metrics)

Reviewer: <img width="1501" alt="Screenshot 2023-02-02 at 6 33 18 PM" src="https://user-images.githubusercontent.com/116072777/216332552-d52211fb-3918-461d-81ae-211e3d46c5b1.png">

> 5. While configuring the cloud trail log source, following Field can be added in the source:

Reviewer: ```suggestion
5. While configuring the cloud trail log source, the following field can be added to the source:
```

AWS Lambda provides Provisoned Concurrency for greater control over the start up time for Lambda functions. When enabled, Provisioned Concurrency keeps functions initialized and hyper-ready to respond in double-digit milliseconds. AWS Lambda provides additional metrics for provisioned concurrency with CloudWatch.

> To collect these metrics in Sumo Logic, do the following:

Reviewer: This comes under the heading "Collect Amazon Cloud Watch metrics," but we're redirecting to the same section. I'm a little confused. Do you think we should move the "To collect..." paragraph before the heading?

Author: Once the doc is released, it will redirect to the above sub-section (Collect Amazon CloudWatch Metrics). Moving it above the current title will place it in the same section it's pointing to.

AWS Lambda provides Provisoned Concurrency for greater control over the start up time for Lambda functions. When enabled, Provisioned Concurrency keeps functions initialized and hyper-ready to respond in double-digit milliseconds. AWS Lambda provides additional metrics for provisioned concurrency with CloudWatch.

To collect the metrics in Sumo Logic, follow the steps below:

> 1. Complete this [Step](#collect-amazon-cloudwatch-metrics).

Reviewer: ```suggestion
1. Jump to the [Collect Amazon Cloudwatch Metrics](#collect-amazon-cloudwatch-metrics) section and complete the steps as described.
```

> To collect these metrics in Sumo Logic, do the following:

Reviewer: ```suggestion
To collect the metrics in Sumo Logic, follow the steps below:
```

To collect these metrics in Sumo Logic, do the following:

> 1. Complete [Step](#collect-amazon-cloudwatch-metrics).

Reviewer: There should be a number for the step.

Author: There is no number to the step, as it's a sub-section within the doc. I have added an article for the sentence to be complete.

## Installing the AWS Lambda App

> Now that you have set up collection for AWS Lambda, install the Sumo Logic App to use the pre-configured searches and [dashboards](#viewing-aws-lambda-dashboards) that provide visibility into your environment for real-time analysis of overall usage. To install the app:

Reviewer: ```suggestion
Now that you have set up collection for AWS Lambda, install the Sumo Logic App to use the pre-configured searches and [dashboards](#viewing-aws-lambda-dashboards) that provide visibility into your environment for real-time analysis of overall usage. To install the app, follow the steps below:
```
